Entity/ContentType.php

```diff
@@ -420,6 +413,27 @@ class ContentType extends JsonDeserializer implements \JsonSerializable
      */
     protected $createLinkDisplayRole = 'ROLE_USER';
 
+    /**
+     * @var string[]
+     *
+     * @ORM\Column(name="version_tags", type="json_array", nullable=true)
+     */
+    protected $versionTags = [];
```

Comment:
Why multiple version tags?

Reply:
Its on the contentType: Major, Minor, ..
On the publish you can only select Silent or ONE of the version tags